chain/gen/gen.go

```diff
@@ -276,8 +274,13 @@ func CarWalkFunc(nd format.Node) (out []*format.Link, err error) {
 }
 
 func (cg *ChainGen) nextBlockProof(ctx context.Context, pts *types.TipSet, m address.Address, round int64) (*types.EPostProof, *types.Ticket, error) {
+	mc := &mca{w: cg.w, sm: cg.sm}
 
-	lastTicket := pts.MinTicket()
+	// TODO: REVIEW: Am I doing this correctly?
+	ticketRand, err := mc.ChainGetRandomness(ctx, pts.Key(), crypto.DomainSeparationTag_TicketProduction, pts.Height(), m.Bytes())
```

Comment:
i heard some rumblings about people wanting the inputs to randomness to be a cbor object. though i'm not sure if thats supposed to be the object here, or the one we pass into computeVRF